services/src/contexts/migrations/database_migration.rs

```diff
@@ -106,24 +108,24 @@ where
             }
         );
 
-        info!(target: prefix, "Applying migration: {}", migration.version());
+        info!("Applying migration: {}", migration.version());
 
-        let tx = conn.build_transaction().start().await?;
+        let tx = connection.build_transaction().start().await?;
 
         migration.migrate(&tx).await?;
 
-        let stmt = tx
-            .prepare("UPDATE geoengine SET database_version = $1")
-            .await?;
-
-        tx.execute(&stmt, &[&migration.version()]).await?;
+        tx.execute(
+            "UPDATE geoengine SET database_version = $1",
```

Comment:
this is not necessary if it is created from scratch and the current_schema.sql already set it.

Reply:
cf. above